docs/content/en/reference/cli/restart.md

```diff
@@ -1,6 +1,9 @@
 ## azk restart
 
-  Stops either all instances of the systems in the current `Azkfile.js` or the one specified and starts them back again. If an error occurs during reboot, all systems will be stopped.
+  Restart works by memorizing the current status of the systems in the current `Azkfile`, stops them and then get them back online again.
+  If a `[system]` is specified with the command, `azk` will just restart the specified system and it's dependencies.
```

Comment:
@slobo you're right, to restore the dependencies they must be passed in the form of a list: `azk restart dependency_system,system`